modules/compliance-unapplying.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-remediation.adoc
+
+[id="compliance-unapplying_{context}"]
+= Unapplying a remediation
+It might be required to unapply a remediation that was previously applied.
+
+.Procedure
+. Toggle the flag to `false`:
++
+[source,terminal]
+----
+$ oc patch complianceremediations/sysctl-net-ipv4-conf-all-accept-redirects --patch '{"spec":{"apply":false}}' --type=merge
+----
```

Comment:
should be `$ oc patch complianceremediations/<scan_name>-sysctl-net-ipv4-conf-all-accept-redirects`